chain/types/ethtypes/eth_transactions.go

```diff
@@ -253,6 +253,10 @@ func getFilecoinMethodInfo(recipient *EthAddress, input []byte) (*methodInfo, er
 	var to address.Address
 	var method abi.MethodNum
 
+	var zeroEthAddr EthAddress
+	if recipient != nil && *recipient == zeroEthAddr {
+		return nil, fmt.Errorf("zero address recipient disallowed, please set `To` to null")
+	}
```

Comment:
This is used inside message validation (now) and is therefore consensus critical. Given that the user _can_ send to this address (it is a valid account), we're not allowed to do this.

Reply:
FYI @rvagg this was a pretty bad mistake on my part.  If I'd merged this and we hadn't thought more about it we would have opened up a fork.  So I guess the lesson for you here is to trust me less.